tasks/task03/ Word_stats.py

```diff
@@ -0,0 +1,16 @@
+import re
+texts = ["Hello, World!", "The world is mine", "Hello, how are you?"]
+dictw = {}
+i = 0
+for istr in texts:
```

Comment:
There is a function enumerate(collection) which yields an iterable of pairs (index, item), it is very useful when you need both item and its index, just like here:

for i, istr in enumerate (texts):